src/typechecker/FStar.TypeChecker.Normalize.fst

```diff
@@ -866,9 +866,10 @@ let is_forall_const cfg (phi : term) : option term =
 let is_extract_as_attr (attr: attribute) : option term =
   let head, args = head_and_args attr in
   match (Subst.compress head).n, args with
-  | Tm_uinst({n=Tm_fvar fv}, [u]), [_; (t, _)]
-        when Syntax.fv_eq_lid fv PC.extract_as_lid ->
-      Some t
+  | Tm_fvar fv, [t, _] when Syntax.fv_eq_lid fv PC.extract_as_lid ->
```

Comment:
Not a big deal but probably better to avoid these back-and-forth changes within a PR.

Reply:
I was going by the [contribution guidelines](https://github.com/FStarLang/FStar/blob/master/CONTRIBUTING.md#merge-vs-rebase).  We should probably update them now.